bootstrap/redis/producer/events.go

```diff
@@ -51,7 +51,7 @@ type createConfigEvent struct {
 
 func (cce createConfigEvent) encode() map[string]interface{} {
 	return map[string]interface{}{
-		"id":          cce.mfThing,
+		"thing_id":    cce.mfThing,
```

Comment:
Why do we use snake_case here? Isn't cammelCase something that is Go standard?

Reply:
This is not `go` event but Redis event. If events were in JSON format, we would use underscore (AFAIK we already use it for JSON in our API).

Reply:
I've changed `id` to `thing_id` just to be in sync with Things service. I think it's also more precise. There is no other reason for this change.